### ISSUE TITLE
Character Profiles

### ISSUE THREAD

**Issue by Calysix** [Bloodmallet/bloodmallet_web_frontend#62]
All simulations are based on the character profiles.  Many people trust the character profiles are similar to what they play and rarely check them when looking at the results of the simulations.
There are specs in the game that should have more than one character profile.  I know bloodmallet is looking to use the highest DPS simulation as a dividing line but it doesn't help the viewer for a couple of reasons.
Example 1:  Frost Death Knight has 2 very distinct playstyles that are popular.  Some may even say 3-4 playstyles.  The existing character profile will show results for Breath of Sindragosa.  A large portion of the Frost Death Knight player base are strongly against using that talent and play Obliteration.  None of the charts help you understand how Obliteration would perform.

A more glaring character profile is the existing character profile for Destruction Warlock.  I would dare say no Destruction Warlock is running Soul Fire and Internal Combustion as the Bloodmallet character profile is set for.

Warcraft logs and https://mplus.subcreation.net/ show what people are playing in current content.  You have a clear idea of what character profiles should look like.  That Destruction Warlock profile won't help anyone because no one plays with those talents.  Those talents won't become viable til the end of the expansion when there are enough secondary stats to support it.

In conclusion, the character profiles should be closer to profiles that players are actually using and if a spec has more than one distinct playing style then those distinct styles should be represented.

I know that is a considerable amount of work to be done, but it is a more accurate representation of what players are looking for.

**Comment by Bloodmallet** [comment]
Greetings,
thank you for reaching out and laying out your issue. I understand your concern and was spending a lot of time thinking about the very same issue you describe before setting up the website. Adding more base-profiles than one per spec will increase the necessary simulation power accordingly. Therefore I can't do that. I recognized this limitation and decided to write tech to enable patrons to generate their custom charts. Those use the submitted profile, thus different talent setups, legendaries, you name it.

I still hope to be able to catch all these profiles some day, but there is no ETA. Especially because I'd probably need another source of income (e.g. ads) to power that calculation need. I'm glad to not have to spam everyone with `Berty Bott's Beans` ads so far while keeping the website afloat.

**Comment by Calysix** [comment]
Thank you for the response.  I understand the issue.  I understand Bloodmallet doesn't want to become Raidbots 2.0 but maybe there could be a way to create a Bloodmallet sub-site that people are willing to pay a monthly fee to do those simulations.I just say that as someone that willingly paid a Patreon fee to Bloodmallet and a sub fee to Raidbots.If people are willing the sub fee could pay for all the advancements you want Bloodmallet to experience.Bloodmallet provides unique simulations to Raidbots or simulations in a form customers find interesting.Maybe you can do a poll to see how many would be interested in something similar.
Again, Thank you for the response.
    On Friday, February 19, 2021, 10:46:07 AM CST, BloodmalletEU <notifications@github.com> wrote:  
 
 


Greetings,
thank you for reaching out and laying out your issue. I understand your concern and was spending a lot of time thinking about the very same issue you describe before setting up the website. Adding more base-profiles than one per spec will increase the necessary simulation power accordingly. Therefore I can't do that. I recognized this limitation and decided to write tech to enable patrons to generate their custom charts. Those use the submitted profile, thus different talent setups, legendaries, you name it.

I still hope to be able to catch all these profiles some day, but there is no ETA. Especially because I'd probably need another source of income (e.g. ads) to power that calculation need. I'm glad to not have to spam everyone with Berty Bott's Beans ads so far while keeping the website afloat.

**Comment by Bloodmallet** [comment]
Thank you for the exchange. I'm closing the thread for now to clean up the repository. But please feel free to reopen it with new thoughts.
